narwhals/_duckdb/expr.py

```diff
@@ -8,6 +8,7 @@
 from typing import Sequence
 from typing import cast
 
+import duckdb
```

Comment:
Should we follow the same pattern as below and directly import `from duckdb import SQLExpression`?